plansys2_problem_expert/launch/problem_expert_launch.py

```diff
@@ -30,28 +31,37 @@ def generate_launch_description():
         'plansys2_domain_expert/test/pddl/domain_simple.pddl',
         description='PDDL Model file')
 
+    declare_problem_file_cmd = DeclareLaunchArgument(
+        'problem_file',
+        default_value='src/ros2_planning_system/'
```

Comment:
The default value should be void string (""), which means that there is not problem defined.

Reply:
done